tests/frontend/unit/DpAutofillSubmitterData.json

```diff
@@ -0,0 +1,496 @@
+[
```

Comment:
Should we move this file to a `fixtures` subdirectory?

Reply:
moved in ebea42bf8028deab62c33b12c886a13acfdec18f